### ISSUE TITLE
instance.subnet displays wrong subnets

### ISSUE THREAD

**Issue by sdouche** [wallix/awless#95]
List of subnets of my new account:
```
$ awless --aws-profile MYACCOUNT list subnets

|      ID ▲       | NAME |      CIDR      |    ZONE    | DEFAULT |     VPC      | PUBLIC |   STATE   |
|-----------------|------|----------------|------------|---------|--------------|--------|-----------|
| subnet-43cafb1b |      | 172.31.32.0/20 | eu-west-1a | true    | vpc-3f9f7758 | true   | available |
| subnet-7abe4a1d |      | 172.31.0.0/20  | eu-west-1b | true    | vpc-3f9f7758 | true   | available |
| subnet-c6e91f8f |      | 172.31.16.0/20 | eu-west-1c | true    | vpc-3f9f7758 | true   | available |
```

Awless display wrong subnets (it's another account):
```
$ awless --aws-profile MYACCOUNT create instance keypair=MYKEY
Please specify (Ctrl+C to quit, Tab for completion):
instance.name? test
instance.subnet?
subnet-2c360c74      subnet-34e41a53      subnet-b24bb0fb

**Comment by fxaguessy** [comment]
Thanks for the bug report. 
This is very probably because the last local model that was stored locally was using a different AWS profile. As a result, the subnets proposed by completion are those of this other profile.
We will fix that after 0.1.0 by improving the way we store local models. Actually, we have the same problem if you switch region using, for example, the --aws-region flag.

**Comment by simcap** [comment]
True after 0.1.0, we will be able to store multi-{region/profile} simultaneously, which will solve the problem.

As for now, it seems the use of `--aws-profile` as a quick handy flag to be in the correct profile (before creating resource) might be due to the fact that it is currently a pain to set anther profile and region through the cumbersome command `awless config set aws.profile my-profile`

Maybe an `awless switch` specific for quickly setting profile/region could come in handy. On this command an `--no-sync` flag could be used to not trigger sync when switching profile.

If doing an `awless switch` command before next week, **smart completion** of your (often used/most used, like a stack) profile/region would be paramount!

**Comment by fxaguessy** [comment]
This is fixed (since `0.1.1`) and completion now works as expected.
